foundations/introduction/motivation_and_mindset.md

```diff
@@ -166,9 +149,25 @@ The Odin Project does not recommend taking a lot of notes throughout your web de
 
 Learning any new skill is a journey full of speed bumps and obstacles to be overcome. We hope that the principles laid out here will put you in a much better position to succeed and get the most out of The Odin Project.
 
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. To give your motivation a bit of a boost, read about the success of others in our discord [odin-success-stories](https://discord.com/channels/505093832157691914/1089990025162260570) channel (You need to [join the Discord server](https://discord.gg/fbFCkYabZB) first in order to see the channel).
+1. To learn more about the growth mindset, check out these resources:
+  - [Believe you can get better](https://www.ted.com/talks/carol_dweck_the_power_of_believing_that_you_can_improve)
+  - [Grit](https://ted.com/talks/angela_lee_duckworth_grit_the_power_of_passion_and_perseverance)
+  - [You can learn anything](https://www.khanacademy.org/college-careers-more/talks-and-interviews/talks-and-interviews-unit/conversations-with-sal/a/the-learning-myth-why-ill-never-tell-my-son-hes-smart)
+1. To learn more about the best ways to learn, read [learning how to learn on Coursera](https://www.coursera.org/learn/learning-how-to-learn).
```

Comment:
Nit: While we're here, doing the full course could be a lengthy distraction and unnecessary for learners. I think it should be optional and in the additional resources. We could replace it with a quick summary video instead:

```suggestion
1. To learn more about the best ways to learn, watch this summary of [Coursera's learning how to learn course](https://www.youtube.com/watch?v=O96fE1E-rf8). The full course is linked in the additional resources.
```